src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -167,6 +167,14 @@ class TrialWaveFunction : public MPIObjectBase
                         ParticleSet::ParticleGradient_t& fixedG,
                         ParticleSet::ParticleLaplacian_t& fixedL);
 
+  static void flex_evaluateDeltaLog(const RefVector<TrialWaveFunction>& wf_list,
```

Comment:
Document these functions evaluateDeltaLog and flex_evaluateDeltaLog to your best knowledge please.

Reply:
Can we move the documentation from cpp to to header?